scripts/local-test.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+const path = require('path');
+const semver = require('semver');
+const cp = require('child_process');
+const packageJson = require('../package.json');
+
+function install() {
+    // bump up
+    const version = new semver.SemVer(packageJson.version)
+    version.inc('patch');
+    packageJson.version = version.version;
+    fs.writeFileSync(path.join(__dirname, '../package.json'), JSON.stringify(packageJson, null, '\t') + "\n");
+
+    // install
+    cp.execSync('yarn package')
+    cp.execSync(`code --install-extension gitpod-desktop-${version.version}.vsix --force`)
```

Comment:
but why? we have a debug target? 🤔

Reply:
I need only one daemon (with current code base) to be started. And to make it like the env that user using